docs/source/schemas/organization.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Organization
+description: Scaling your Apollo Server from a single file to your entire team
+---
+
+## Prerequisites
+
+* Understanding of GraphQL types
+* Resolvers
+
+## Overview
+
+The schema contains the information to define all requests that the client can request from an instance of Apollo Server along with the resolvers necessary to route the requests to retrieve data. For most applications, the schema type definitions are be placed in a single file along side the resolvers. Placing reolvers in the same file as their accompanying type definitions is the best way to organize the code, since it enables developers to locate and modify the two inter-dependent portions.
+
+Sometimes production servers contain a typeDefs string of over a thousand lines, which makes it difficult to maintain a file containing the resolvers as well. For applications with multiple teams or product domains, this section describes an example application and methods for organizing types and resolvers to make a large instance more modular. The separation between types should follow real-world domains, for example movies vs books, rather than the backend organization. To facilitate this organization, common practice is to create a data model layer that enables resolvers across domains to request data from a common interface. A data layer further enables the different schema domains to share data, such as a user profile.
+
+Along with modularizing large schemas, GraphQL enables schemas to include documentation inline that is viewable in GraphiQL.
+
+## Example Application
+
+The application contains a schema, resolvers with fake data, and the Apollo Server start code.
+
+### Types
+
+When using `apollo-server`, the schema is defined as a string in the [GraphQL SDL]().
+
+```js
+const typeDefs = `
+  type Author {
+    id: Int!
+    firstName: String
+    lastName: String
+    """
+    the list of Posts by this author
+    """
+    posts: [Post]
+  }
+
+  type Post {
+    id: Int!
+    title: String
+    author: Author
+    votes: Int
+  }
+
+  # the schema allows the following query:
+  type Query {
+    posts: [Post]
+    author(id: Int!): Author
+  }
+
+  # this schema allows the following mutation:
+  type Mutation {
+    upvotePost (
+      postId: Int!
+    ): Post
+  }
+`;
+```
+
+### Resolvers
+
+In the same file as the type defintinos, the resolvers are organized as a nested object that maps type and field names to functions:
```

Comment:
> In the same file as the type defintinos,

defintinos => definitions

Reply:
we're eventually going to delete this file :)